app/views/provider_interface/organisations/settings.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= content_for :browser_title, 'Organisation settings' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Organisation settings</h1>
+    <ul class="govuk-list govuk-list--spaced">
+      <% if current_provider_user.authorisation.can_manage_users_for_at_least_one_provider? %>
+        <li>
+          <%= govuk_link_to t('page_titles.provider.users'), provider_interface_provider_users_path %>
+        </li>
+      <% end %>
+
+      <% if current_provider_user.authorisation.can_manage_organisations_for_at_least_one_setup_provider? %>
+        <li>
+          <%= govuk_link_to t('page_titles.provider.org_permissions'), '#not-linked-yet' %>
```

Comment:
Could we just link to the current page rather than nowhere?

Reply:
Using something a bit more specific allows me to verify the link in the system spec (see `and_i_see_a_link_to_manage_organisations`). It is temporary and behind a feature flag, anyway.